src/Storage/Storage.Management/Storage.Management.csproj

```diff
@@ -15,9 +15,9 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Storage" Version="12.0.0" />
-    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="10.0.3" />
-    <PackageReference Include="Microsoft.Azure.Storage.File" Version="10.0.3" />
-    <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="10.0.3" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.0.0" />
```

Comment:
This is a major version change.  ARe there breaking changes in the underlying libraries?

Reply:
@markcowl 
You can find the changes for this release in: https://github.com/Azure/azure-storage-net/releases

There are some APIs (like: Copyblob, SetBlobTier) has parameter breaking, since new parameter added before existing parameter, this will make the code write with old SDK build fail. But this won't take PSH breaking, and I have modify PSH code to absorb the breaking and make build pass.

Reply:
But, Storage has some scenarios in which you sak your users to call methods from the types returned from your cmdlets.  DO any of these scenarios have breaking changes?

Reply:
The old and new DMlib depend different version of XSCL, and the old and new XSCL has different namespace, so there will be not problem for customer to call the XSCL API in Powershell, since the namespace of old and new XSCL type are different.